Library/Sources/Common/VK.swift

```diff
@@ -26,6 +26,11 @@ public final class VK {
         
         return sessionsHolder
     }
+    
+    /// Free up all resources
```

Comment:
/// Free up all SwiftyVK's resources and release memory
/// Call it only if you won't interact with the library anymore 
/// If you'll need to work with the library again after calling of this method, you should call setUp